source/Calamari.Common/Features/Scripting/ScriptWrapperPriorities.cs

```diff
@@ -7,6 +7,16 @@ namespace Calamari.Common.Features.Scripting
     /// </summary>
     public static class ScriptWrapperPriorities
     {
+        /// <summary>
+        /// The priority for script wrappers that configure kubernetes authentication
+        /// </summary>
+        public const int KubernetesContextPriority = 1002;
```

Comment:
Should this be `ScriptWrapperPriorities.CloudAuthenticationPriority + 2` ?

Reply:
They are the same value, right? Since all priorities are now defined in the same class next to each other, later developers should be able to pretty easily figure out the order when reading this class.